### PR TITLE
PROD-295/campaign-detail-cleanup

### PR DESCRIPTION
**Pull request by yash431garg**
- Description
1. Added route back to campaign on specific campaign page
2. Udated campaign image as fallback image and set default to deck image.
- What are the steps to test that this code is working?
1. Redirecting back to campaigns from /campaigns/{campaignsId}
2. Verifying decks images on /campaigns/{campaignsId}

### CODE REVIEW

File: app/application/decks/[id]/page.tsx [hunk 1]
```diff
@@ -20,7 +20,7 @@ export default async function Page({ params: { id } }: PageProps) {
     <div className="h-full pt-3 pb-4">
       {deck?.questions.length === 0 || deck === null ? (
         <NoQuestionsCard variant={"regular-deck"} nextDeckId={nextDeck?.id} />
-      ) : deck?.questions && deck?.questions?.length > 0 && deck?.deckInfo ? (
+      ) : deck?.questions && deck?.questions?.length > 0 && deck?.deckInfo? (
```
Comment (Yihwan) [inline app/application/decks/[id]/page.tsx:23]: is this supposed to be a ternary?

**Comment by Yihwan** [comment]
the preview seems to work as expcted defer cr to @marvinmarnold

File: app/application/decks/[id]/page.tsx (hunk 1 above)
Comment (marvinmarnold) [reply app/application/decks/[id]/page.tsx:23]: Agreed this is weird. This same change appears in other of your recent PRs. Do you have some weird linting/

File: app/campaigns/[id]/page.tsx [hunk 2]
```diff
@@ -62,7 +61,7 @@ const CampaignPage = async ({ params: { id } }: PageProps) => {
             }
             deckQuestions={deck.deckQuestions.map((dq) => dq.question)}
             deckName={deck.deck}
-            imageUrl={campaign.image}
+            imageUrl={deck.imageUrl ? deck.imageUrl : campaign.image}
```
[the remark this answers is not in the document]
Comment (yash431garg) [reply app/campaigns/[id]/page.tsx:64]: Will keep in mind from upcoming PRs

**Review by marvinmarnold — CHANGES_REQUESTED** [review]

**Pull request closed without merging**